inc/osvr/ClientKit/ContextC.h

```diff
@@ -49,7 +49,15 @@ OSVR_EXTERN_C_BEGIN
 
 /** @addtogroup ClientKit
     @{
-*/
+    */
+
+/** @brief Ensures the OSVR server process/thread is running. Call once per process.
+ */
+ OSVR_CLIENTKIT_EXPORT OSVR_ReturnCode osvrServerAutoStart();
```

Comment:
So we are decided then on this API? Do we need any parameters here? Do we need to document what return codes mean? (or more likely, what they don't mean? like, we explicitly state that they mean nothing about whether or not we had to start the server...)

Also an indication that how exactly the server is started, if at all, is a platform-specific implementation detail

Also the name of the function - Think the rest of the general clientkit apis are `osvrClient` - maybe `osvrClientAttemptServerAutoStart()` ?